test/sequential/test-benchmark-http.js

```diff
@@ -20,6 +20,9 @@ const path = require('path');
 
 const runjs = path.join(__dirname, '..', '..', 'benchmark', 'run.js');
 
+const env = process.env;
```

Comment:
maybe
`const env = Object.assign({}, process.env, { NODEJS_BENCHMARK_ZERO_ALLOWED: 1 })`
For better hygiene